app/controllers/concerns/default_page_content.rb

```diff
@@ -0,0 +1,12 @@
+module DefaultPageContent
+  extend ActiveSupport::Concern
+
+  included do 
+  	before_action :set_page_defaults
+  end
+
+  def set_page_defaults
+  	@page_title = "Note-Keeping"
+  	@seo_keywords = "Kiprosh Note-Keeping App"
```

Comment:
follow 2-space convention
```suggestion
  @page_title = "Note-Keeping"
  @seo_keywords = "Kiprosh Note-Keeping App"
```